src/main/resources/flyway/V1__Initial_model.sql

```diff
@@ -9,9 +9,9 @@ CREATE TABLE emails2send (
 CREATE TABLE library (
     "uid" VARCHAR(36) UNIQUE NOT NULL,
     "name" TEXT NOT NULL,
-    "author" TEXT NOT NULL,
+    "author" TEXT ,
```

Comment:
Have you updated also the Slick schema definition ? (just asking not to miss it)

Reply:
Yes